website/client/components/groups/questSidebarSection.vue

```diff
@@ -54,10 +54,12 @@ div
             .row.boss-details
                 .col-6
                   span.float-left
-                    | {{parseFloat(group.quest.progress.hp).toFixed(2)}} / {{parseFloat(questData.boss.hp).toFixed(2)}}
+                    | {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }} / {{ parseFloat(questData.boss.hp).toFixed(2) }}
+                    // current boss hp uses ceil so you don't underestimate damage needed to end quest
                 .col-6(v-if='userIsOnQuest')
-                  // @TODO: Why do we not sync quset progress on the group doc? Each user could have different progress
-                  span.float-right {{parseFloat(user.party.quest.progress.up).toFixed(1) || 0}} pending damage
+                  // @TODO: Why do we not sync quest progress on the group doc? Each user could have different progress.
+                  span.float-right {{ Math.floor(parseFloat(user.party.quest.progress.up) * 10) / 10 }} {{ $t('pendingDamage') }}
```

Comment:
could this use the floor filter we already have in client/filters/floor? Maybe with an optional parameter to use *10 instead of *100

Also I don't think that parseFloat is necessary, it's already a number

Reply:
@paglias  Since I made this PR, the `pendingDamage` locales text has been implemented to contain the damage variable in it:
https://github.com/HabitRPG/habitica/blob/70500d7c98d67f1edd4a91aa26e176c5952ebae8/website/common/locales/en/quests.json#L125

So I could change the pending damage line in my code to be like this:
`span.float-right {{ $t('pendingDamage', {damage: user.party.quest.progress.up}) }}`
and use the float function in it (how would I do that though?)

Or I could add a different text string, `pendingDamage1` or something, and use code like this:
`span.float-right {{ user.party.quest.progress.up | float(10) }} {{ $t('pendingDamage1') }}`
Which would you prefer?

Reply:
Let's use the second option

Reply:
That's done - the floor function is now used instead of calling Math.floor. I also made the same change to the user's pending damage in the World Boss box in the Tavern.

This is ready for review again.